src/analyses/global_may_alias.cpp

```diff
@@ -11,6 +11,11 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "global_may_alias.h"
 
+/// Populate list of aliases for a given identifier in
+/// an LHS expression.
```

Comment:
Maybe we should avoid the use of left-hand side/LHS and right-hand side and instead be explicit and take about contexts where an object is being written to vs objects are only being read?